localstack-core/localstack/utils/docker_utils.py

```diff
@@ -147,23 +146,22 @@ def container_ports_can_be_bound(
         port_mappings.add(port.port, port.port, protocol=port.protocol)
     try:
         result = DOCKER_CLIENT.run_container(
-            _get_ports_check_docker_image(),
-            entrypoint="sh",
-            command=["-c", "echo test123"],
+            "bash",
+            command=["echo", "test123"],
             ports=port_mappings,
             remove=True,
         )
     except Exception as e:
-        if "port is already allocated" not in str(e) and "address already in use" not in str(e):
+        if "address already in use" not in str(e):
             LOG.warning(
-                "Unexpected error when attempting to determine container port status: %s", e
+                "Unexpected error when attempting to determine container port status", exc_info=e
```

Comment:
If we do see the message, it would be nice to see the full stacktrace to debug what's going on